### PR TITLE
docs: added summary section to contributing docs

### PR DESCRIPTION
**Pull request by dreamorosi**
## Description of your changes

Following some confusion from community contributors and maintainers alike this PR adds a terse TL;DR; (aka too long; didn't read) section at the top of the `CONTRIBUTING.md` document to allow new contributors more easily get started.

### How to verify this change

Check changes in the Markdown file, preview below:
image

### Related issues, RFCs

N/A

### PR status

***Is this ready for review?:*** YES 
***Is it a breaking change?:*** NO

## Checklist

- [x] [My changes meet the tenets criteria](https://awslabs.github.io/aws-lambda-powertools-python/#tenets)
- [x] I have performed a *self-review* of my own code
- [x] I have made corresponding changes to the *documentation*
- [x] My changes generate *no new warnings*
- [x] The *code coverage* hasn't decreased
- [x] The PR title follows the [conventional commit semantics](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/main/.github/semantic.yml#L2)

### Breaking change checklist

N/A

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,6 +32,12 @@ reported the issue. Please try to include as much information as you can. Detail
 
 Contributions via pull requests are much appreciated.
 
+### TL;DR;
```
Comment (ijemmy) [inline CONTRIBUTING.md:35]:
```suggestion
### Summary
```

TLDR gives me the feeling that I don't have to read this. But it's still important to read through at some point.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -32,6 +32,12 @@ reported the issue. Please try to include as much information as you can. Detail
...
+* This project uses `node@14.x` and `npm@8.x` for development (see [Setup](#setup))
+* Before opening a PR, find an existing issue or open a new one to discuss the proposed change (we would hate it for you to have to waste time)
```
Comment (ijemmy) [inline CONTRIBUTING.md:37]:
```suggestion
* Before opening a PR, find an existing issue or open a new one to discuss the proposed change. PR without an issue or discussion has high risk of being rejected. We don't want to you to waste time. And it is very uncomfortable for us to reject a PR
```

Not sure if it's too long?

**Review by ijemmy — CHANGES_REQUESTED** [review]
Not sure about the 2nd proposed change. May be too long?

But I do feel that we shouldn't use "TL;DR;" here. It is like telling them that it is ok to not read.

**Comment by dreamorosi** [comment]
> Not sure about the 2nd proposed change. May be too long?
> 
> But I do feel that we shouldn't use "TL;DR;" here. It is like telling them that it is ok to not read.

I like both your proposals better, thanks for the suggestions. Updated also the title of the PR.

**Review by ijemmy — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -32,6 +32,12 @@ reported the issue. Please try to include as much information as you can. Detail
 
 Contributions via pull requests are much appreciated.
 
+### Summary
+* This project uses `node@14.x` and `npm@8.x` for development (see [Setup](#setup))
+* Before opening a PR, find an existing issue or open a new one to discuss the proposed change. PR without an issue or discussion has high risk of being rejected. We don't want to you to waste time. And it is very uncomfortable for us to reject a PR
```
Comment (saragerion) [inline CONTRIBUTING.md:37]: Suggestion(minor): different phrasing. 

```suggestion
* Before opening a Pull Request, please find the existing related issue or open a new one to discuss the proposed change. A PR without a related issue or discussion has an high risk of being rejected. We are very appreciative and thankful for your time and efforts and we want to make sure they are not wasted.
```

**Review by saragerion — COMMENTED** [review]
Added a minor suggestion :) thanksss

**Review by saragerion — APPROVED** [review]

**Review by ijemmy — APPROVED** [review]
